### PR TITLE
chore: bump ChainTxData for mainnet and testnet

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
mainnet
```
getchaintxstats
{
  "time": 1660074878,
  "txcount": 43702293,
  "window_final_block_hash": "0000000000000002ee5a0d2caa3f78cd630ece1a12ce74f7a8146eb6689b1b66",
  "window_final_block_height": 1718597,
  "window_block_count": 17280,
  "window_tx_count": 476084,
  "window_interval": 2724994,
  "txrate": 0.174710109453452
}
```

testnet
```
> $ ./src/dash-cli --testnet getchaintxstats                                                               [±chore/bump-min-work-assume-valid ●]
{
  "time": 1659319535,
  "txcount": 5577222,
  "window_final_block_hash": "00000c9d1544ca21b5a2070c61cab65b2d7ef7aa1afac94a949e65cecb0b7d0f",
  "window_final_block_height": 771537,
  "window_block_count": 17280,
  "window_tx_count": 40775,
  "window_interval": 2533438,
  "txrate": 0.01609472977037528
}
```

### CODE REVIEW

File: src/chainparams.cpp [hunk 1]
```diff
@@ -558,10 +558,10 @@ class CTestNetParams : public CChainParams {
         };
 
         chainTxData = ChainTxData{
-            1617874832, // * UNIX timestamp of last known number of transactions (Block 477483)
-            4926985,    // * total number of transactions between genesis and that timestamp
+            1659319535, // * UNIX timestamp of last known number of transactions (Block 771537)
+            5577222,    // * total number of transactions between genesis and that timestamp
                         //   (the tx=... number in the ChainStateFlushed debug.log lines)
-            0.01        // * estimated number of transactions per second after that timestamp
+            0.016       // * estimated number of transactions per second after that timestamp
```
Comment (UdjinM6) [inline src/chainparams.cpp:564]:
```
> dash-cli getchaintxstats 17280 00000c9d1544ca21b5a2070c61cab65b2d7ef7aa1afac94a949e65cecb0b7d0f
error code: -5
error message:
Block not found
> dash-cli getblockhash 771537
0000028ce7bc90ddaa75703bbe576b8821e470b4b98bbe13be81eb79546e111f
> dash-cli getchaintxstats 17280 0000028ce7bc90ddaa75703bbe576b8821e470b4b98bbe13be81eb79546e111f
{
  "time": 1659215338,
  "txcount": 5579961,
  "window_final_block_hash": "0000028ce7bc90ddaa75703bbe576b8821e470b4b98bbe13be81eb79546e111f",
  "window_final_block_height": 771537,
  "window_block_count": 17280,
  "window_tx_count": 43514,
  "window_interval": 2428572,
  "txrate": 0.01791752519587642
}
```

**Review by UdjinM6 — CHANGES_REQUESTED** [review]
pls see below

File: src/chainparams.cpp (hunk 1 above)
Comment (thephez) [reply src/chainparams.cpp:564]: :point_up_2: My node agrees with Udjin's

**Comment by PastaPastaPasta** [comment]
force pushed

**Review by UdjinM6 — APPROVED** [review]
LGTM, utACK

**Pull request merged**